v63005/outref/gtm8962.txt

```diff
@@ -0,0 +1,16 @@
+# Confirming $ZPIN=$PRINCIPAL, searching for $ZPIN in the output of ZSHOW "I"
+YDB>write:($ZPIN=$PRINCIPAL) "# $ZPIN = $PRINCIPAL"
+# $ZPIN = $PRINCIPAL
+$ZPIN="/dev/pts/3"
```

Comment:
Also I suspect the 3 in /dev/pts/3 will not be the same every time we run this test. It depends on what other terminals are currently in use. So this might need to be TEST_AWKed out.